doc/app/SOCKET_AT_commands.rst

```diff
@@ -911,10 +911,11 @@ Syntax
 * The ``<flags>`` parameter sets the sending behavior.
   It can be set to the following value:
 
-  * ``0`` - No flags set.
-  * ``512`` - Blocks send operation until the request is acknowledged.
-    The request will not return until the send operation is completed by lower layers, or until the timeout given by the AT_SO_SNDTIMEO socket option, is reached.
+  * ``0`` - No flags set. The request is complete when the data is pushed to the modem buffer.
+  * ``512`` - Network acknowledged send.
```

Comment:
I guess we have three options:

1. Change flag 512 behavior from MSG_WAITACK to SEND_NW_ACK_NTF.
2. Add new flag value for SEND_NW_ACK_NTF and keep MSG_WAITACK (512) as is.
3. Add new flag value for SEND_NW_ACK_NTF and remove MSG_WAITACK (512).

My proposal is #3 but I want to confirm this offline from some stakeholders.

Reply:
I'm fine with option `#2` also. I'm just thinking whether it will be one more unused feature but it's really no work for us either.
8192 sounds good as a value.